pkg/server/datastore/sqlstore/sqlstore.go

```diff
@@ -294,7 +294,9 @@ func (ds *Plugin) CreateAttestedNode(ctx context.Context, node *common.AttestedN
 		if err != nil {
 			return err
 		}
-		return createAttestedNodeEvent(tx, node.SpiffeId)
+		return createAttestedNodeEvent(tx, &datastore.AttestedNodeEvent{
```

Comment:
Attested nodes use the same kind of event caching to avoid the same kind of need to periodically poll to maintain cache and memory consistency.  However, there is no user command to create the AttestedNodeEvent (in regular SPIRE Entires, they are created as side-effects of the "spire entry create" command.  So, this "createAttestedNodeEvent()" has to exist to pump the "node entry events" table with items so the same logic used for other non-node entries can be used for node entries.

Reply:
Required for unit tests to create events out of order.